composer.json

```diff
@@ -35,7 +35,7 @@
     "symfony/validator": "^6.4",
     "symfony/webpack-encore-bundle": "^2.1",
     "symfony/yaml": "^6.4",
-    "tiqr/tiqr-server-libphp": "^4.2",
+    "tiqr/tiqr-server-libphp": "^4.3.0-beta1",
```

Comment:
Perhaps first tag a final release of the libphp server? We might not be able to install this if composer is installing with `stable` preferences. Not sure, but this might not chooch :steam_locomotive: